### PR TITLE
FLOE-198: Add UIO to the Feedback Tool Demo

### PR DESCRIPTION
**Pull request by jobara**
Added UIO to the feedback tool demo. 

http://issues.fluidproject.org/browse/FLOE-198

### CODE REVIEW

**Comment by cindyli** [comment]
Some file permissions are not 644.

**Comment by cindyli** [comment]
Click "show display preferences" to open up UIO, the "reset" button overlaps the header text of the demo. The overlapping is worse when the preference "enlarge buttons" is turned on or on a small screen.

File: demos/feedback/html/index.html [hunk 1]
```diff
@@ -56,6 +92,25 @@ <h2 class="gpii-feedbackDemo-page-title">Climate Change Impacts</h2>
...
+            fluid.uiOptions.prefsEditor(".flc-prefsEditor-separatedPanel", {
+                tocTemplate: "../../../src/lib/infusion/components/tableOfContents/html/TableOfContents.html",
+                templatePrefix: "../../../src/lib/infusion/framework/preferences/html/",
+                messagePrefix: "../../../src/lib/infusion/framework/preferences/messages/",
+                components: {
+                    prefsEditorLoader: {
+                        options: {
+                            iframeRenderer: {
+                                markupProps: {
+                                    "class": "flc-iframe",
```
Comment (cindyli) [inline demos/feedback/html/index.html]: Can this line be removed since the value is the default one that's already defined in prefs framework?

**Comment by cindyli** [comment]
When high (or low) contrast theme is applied, the circles around preference buttons on the feedback top toolbar disappears, but the circles for happy/sad face and request buttons have color changed properly.

**Comment by cindyli** [comment]
When text size increases, the size of the video container on the feedback demo page doesn't increase accordingly.

**Comment by cindyli** [comment]
1. Turn on "table of contents" in UIO;
2. Reload the page;
3. Open up firebug console; 
4. Click "show display preferences" to open up UIO.

A javascript error occurs:
"NetworkError: 404 Not Found - http://localhost/metadata/demos/components/tableOfContents/html/TableOfContents.html?_=1405452893640"

Seems the UIO iframe panel is looking for table of content template at a wrong location. This template is inside the infusion lib - src/lib/infusion/components/tableOfContents/html/TableOfContents.html

**Comment by jobara** [comment]
I talked to Dana and Joanna about the Feedback Tool design and UIO. They had some suggestions which relate to the comments from @cindyli above. 
- The display preferences button should be smaller (see: [outside-in](http://outside-in.idrc.ocadu.ca/) site)
- Joanna will provide mockups for the preference icons when contrast themes are applied
- The video element should scale with the text
- The Feedback Tool should push down the page content if the toolbar wraps and expands.

**Comment by jobara** [comment]
The issue that @cindyli described with the ToC template above was an issue in Infusion. http://issues.fluidproject.org/browse/FLUID-5474

**Comment by jobara** [comment]
I haven't fixed the issue of the feedback tool wrapping at small screens. I think we may want to provide a responsive design for this instead.

**Comment by cindyli** [comment]
@jobara, thanks for fixing those. Now the icons look great even with high contrast themes.

A minor issue I noticed is if you use "Verdana" font, maximize line spacing, enlarge buttons, the reset button would overlap the header text.

**Comment by jobara** [comment]
@cindyli ready for another round of review. I've addressed the overlap issue when line spacing is set. I've also filed a jira for the feedback tool overlapping the header and the UIO button when the icons don't fit on a single row. http://issues.fluidproject.org/browse/FLOE-200

**Comment by cindyli** [comment]
Merged into the master branch @ 017b8c8163a3433bc3ad6282c4ed9088cbede38b

**Pull request merged**